test/specs/entityImport.js

```diff
@@ -0,0 +1,78 @@
+    });
+    it('01/LP-Should reset all values', () => {
+        reset();
+        expect($(selectors.navigationBar.recycleBinNotification)
+            .isDisplayed()).false;
+    });
+    it('02/LP-Should go to menu Import', () => {
+            $(selectors.entityImport.menuImport).scrollIntoView();
+            $(selectors.entityImport.menuImport).click();
+            expect($(selectors.entityImport.entityTitle).getText())
+                .to.equal(expected.entityImport.headerOfImport);
+        });
+    it('03/LP-Should create new folder', () => {
+        $(selectors.common.createRecordBtn).click();
+        expect($(selectors.entityImport.recordForm).isDisplayed()).true;
+    });
+    it('04/LP-Should create data of new record', () => {
+        $(selectors.entityImport.stringInputValue)
+            .setValue(testData.entityImport.string);
+        $(selectors.entityImport.textInputValue)
+            .setValue(testData.entityImport.text);
+        $(selectors.entityImport.intInputValue)
+            .setValue(testData.entityImport.integer);
+        $(selectors.entityImport.decimalInputValue)
+            .setValue(testData.entityImport.decimal);
+
+        browser.waitUntil(() => $(selectors.entityImport.stringInputValue)
+            .getValue() === expected.entityImport.string);
+        browser.waitUntil(() => $(selectors.entityImport.textInputValue)
+            .getValue() === expected.entityImport.text);
+        browser.waitUntil(() => $(selectors.entityImport.intInputValue)
+            .getValue() === expected.entityImport.integer);
+        browser.waitUntil(() => $(selectors.entityImport.decimalInputValue)
+            .getValue() === expected.entityImport.decimal);
+    });
+    it('05/LP-Should create new Date', () => {
+        $(selectors.entityImport.date).click;
+        expect($(selectors.entityImport.date).isDisplayed()).to.equal(true);
+    });
+    it('06/LP-Should create new DateTime', () => {
+        $(selectors.entityImport.datetime).click;
+        expect($(selectors.entityImport.datetime).isDisplayed()).to.equal(true);
+    });
+    it('07/LP-Should choose user', () => {
+        $(selectors.entityImport.dropDownUsersMenu).click();
+        $(selectors.entityImport.user).scrollIntoView();
+        $(selectors.entityImport.user).click;
+    });
+    it('08/LP-Should save new record', () => {
+        $(selectors.common.saveBtn).click();
+        expect($(selectors.entityImport.recordForm).isDisplayed()).false;
+    });
+    it('09/LP-Should verify new record is saved', () => {
+        expect($(selectors.entityImport.recordIsVisible).isDisplayed())
+            .to.equal(true);
+    });
+    it('10/LP-Should verify the values in new record', () => {
+        for (let i = 0; i < $(selectors.entityImport.listOfTable).length; i++) {
+            expect($(selectors.entityImport.listOfTable)[i].getText())
+                .to.equal(expected.entityImport.expectedValue[i]);
```

Comment:
in this case you for doesn't work because $(selectors.entityImport.listOfTable).length == undefined
Yeah! everybody did this mistake, I know
```suggestion
        for (let i = 0; i < $$(selectors.entityImport.listOfTable).length; i++) {
            expect($$(selectors.entityImport.listOfTable)[i].getText())
                .to.equal(expected.entityImport.expectedValue[i]);
```

Reply:
ok